modules/nf-core/sentieon/bwaindex/tests/nextflow.config

```
@@ -0,0 +1,9 @@
+env {
+    // NOTE This is how nf-core/sarek users will use Sentieon in real world use
```

```suggestion
    // NOTE This is how nf-core/sarek users will use Sentieon in real world use
```
Not all the world is sarek ;)